ray-operator/controllers/ray/utils/constant.go

```diff
@@ -75,7 +75,10 @@ const (
 	ComponentName = "kuberay-operator"
 
 	// The defaule RayService Identifier.
```

Comment:
Could we use `CRDType` instead? https://github.com/ray-project/kuberay/blob/1634d70915c336a38f3bcc5bb4cc8befe359d49a/ray-operator/controllers/ray/utils/util.go#L36

Reply:
Sure.